BridgeAppSDK/SBABridgeManager.m

```diff
@@ -63,7 +63,7 @@ + (void)setupWithBridgeInfo:(id <SBBBridgeInfoProtocol>)bridgeInfo
     [SBAInfoManager sharedManager].currentParticipant = participant;
     
     // Setup the Bridge study
-    [BridgeSDK setupWithBridgeInfo:bridgeInfo];
+    [BridgeSDK setupWithBridgeInfo:bridgeInfo errorUIDelegate:SBAAppDelegate.shared];
```

Comment:
Assumes that this is setup with `SBAAppDelegate` which is not necessarily the case. Because obj-c cannot inherit from Swift 3.0 classes (dangit), apps that were created with an obj-c based app delegate must implement this stuff themselves. (At least for now until adding an obj-c shared delegate bubbles up in priority).

Would be better to be explicit with the setup like we are doing with the auth delegate.